### PR TITLE
Add progress charts

### LINKED ISSUE #133: Add dashboard for gitcollector and ghsync progress
The goal is to have a dashboard that shows the current download progress for gitcollector and ghsync.
The progress will be taken from tables with total/done columns.

It will also need to have some introductory text that explains how data is being downloaded, how you can refresh the dashboard charts, how you can explore with partial results, etc. cc @src-d/product for this.

### PR DESCRIPTION
**Pull request by se7entyse7en**
Closes #133.

This adds progress charts fro `ghsync` and `gitcollector`.

Screenshot 2019-06-21 at 16 09 28

### CODE REVIEW

**Comment by smacker** [comment]
does it contain cache timeout 0 already or not?

**Comment by se7entyse7en** [comment]
@smacker not yet, and I've just realized that for some reason the id changed from 2 to 3 😕.

**Comment by se7entyse7en** [comment]
@smacker I've set `cache_timeout: 0` and also changed the title of the welcome dashboard as @carlosms asked.

File: srcd/dashboards/gitbase/welcome.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
         "created_by_fk": null,
         "json_metadata": "{\"filter_immune_slices\": [], \"timed_refresh_immune_slices\": [], \"filter_immune_slice_fields\": {}, \"expanded_slices\": {}, \"default_filters\": \"{}\", \"remote_id\": 1, \"import_time\": 1560871295, \"refresh_frequency\": 0}",
         "description": null,
-        "dashboard_title": "Welcome",
+        "dashboard_title": "Overview",
```
Comment (smacker) [inline srcd/dashboards/gitbase/welcome.json]: it's still "welcome" few lines above: `"meta\": {\"text\": \"Welcome\"}, \"type\": \"HEADER\"}`

**Comment by smacker** [comment]
Wait, I'll test it.

**Comment by se7entyse7en** [comment]
@smacker I force-pushed the latest changes. Fixed the title as you said and set timeout to 1. Now refreshing the page upates the chart.

**Comment by smacker** [comment]
It opens wrong dashboard by default now

**Comment by smacker** [comment]
Oh. Not just a wrong one. There is no welcome dashboard at all anymore:
Screenshot 2019-06-21 at 17 48 32

**Comment by se7entyse7en** [comment]
@smacker yup, sorry I forgot to mention in the last push I fixed the id of the dashboard. For some reason when I exported it the id changed 🤷‍♂

**Comment by smacker** [comment]
@se7entyse7en what do you mean? I have the latest changes:
```
smacker in ~/Work/sourced-ui on progress-chart [?]
$ git pull
Already up to date.
Current branch progress-chart is up to date.
```

**Comment by marnovo** [comment]
Can I consider this the latest version with progress bars and use it as reference?

**Pull request merged**